otter/test/test_scheduler.py

```diff
@@ -79,31 +94,32 @@ def _start_service(self):
         self.scheduler_service.running = True
 
     def test_start_service(self):
-        """
-        startService() calls super's startService() and creates SetPartitioner object
-        """
```

Comment:
hmm. is it possible to tweak it to try to reduce the churn? ideally it'd be nice if it kept whatever format was there if it was already PEP257-compliant, which `"""Single-line-docstrings"""`, as well as the

>"""
>balanced
>"""

format. The biggest problem, though, is that this format isn't actually conformant to PEP257, since it puts the closing lines on the same line as the content.